mobile/library/common/http/client.h

```diff
@@ -202,12 +204,15 @@ class Client : public Logger::Loggable<Logger::Id::http> {
     absl::optional<envoy_error> error_;
     bool success_{};
 
-    // Buffered response data when in explicit flow control mode.
+    // Buffered response data when in explicit flow control or buffering chunks.
     Buffer::InstancePtr response_data_;
     ResponseTrailerMapPtr response_trailers_;
     // True if the bridge should operate in explicit flow control mode, and only send
     // data when it is requested by the caller.
     bool explicit_flow_control_{};
+    // If greater than zero, indicates the minimum size of data that should be
+    // sent without end stream.
+    uint64_t min_chunk_size_{};
```

Comment:
nit: this is initialized w/ `{}` whereas on L338, we initialize with `= 0`.  Should we make it consistent?  I personally prefer `= 0` but it seems most places in Envoy prefer `{}`.